src/js/pages/Service.js

```diff
@@ -42,7 +46,11 @@ const Service = ({ service, intl }) => {
   return (
     <div>
       <PageBanner image={image} />
-      <PageBreadcrumbs title={title} order={['theme', 'topic']} theme={theme} topic={topic} />
+      <PageBreadcrumbs
+        grandparent={{...theme, subpath: 'themes'}}
```

Comment:
did we land on having the CMS pass in the full path? If so, did we make a task for that?